packages/manager/modules/pci/src/projects/project/streams/stream/stream.html

```diff
@@ -0,0 +1,95 @@
+</div>
+
+<div class="py-4" data-ui-view>
+    <div class="row">
+        <div class="col-md-4">
+            <oui-tile data-heading="{{ 'pci_projects_project_streams_stream_manage_title' | translate }}">
+                <oui-tile-button data-on-click="$ctrl.editBacklogRetention()"
+                    data-ng-if="$ctrl.stream.isPersistent()">
+                    <span data-translate="pci_projects_project_streams_stream_action_editBacklogRetention_label"></span>
+                </oui-tile-button>
+                <oui-tile-button data-on-click="$ctrl.editReplayRetention()"
+                    data-ng-if="$ctrl.stream.isPersistent()">
+                    <span data-translate="pci_projects_project_streams_stream_action_editReplayRetention_label"></span>
+                </oui-tile-button>
+                <oui-tile-button data-on-click="$ctrl.editThrottling()">
+                    <span data-translate="pci_projects_project_streams_stream_action_editThrottling_label"></span>
+                </oui-tile-button>
+                <oui-tile-button data-on-click="$ctrl.viewSubscriptions()">
+                    <span data-translate="pci_projects_project_streams_stream_action_viewSubscriptions_label"></span>
+                </oui-tile-button>
+                <oui-tile-button data-on-click="$ctrl.regenerateToken()">
+                    <span data-translate="pci_projects_project_streams_stream_action_regenerateToken_label"></span>
+                </oui-tile-button>
+                <oui-tile-button data-on-click="$ctrl.deleteStream()">
+                    <span data-translate="pci_projects_project_streams_stream_action_delete_label"></span>
+                </oui-tile-button>
+            </oui-tile>
+        </div>
+
+        <div class="col-md-4">
+            <oui-tile data-heading="{{:: 'pci_projects_project_streams_stream_info_title' | translate }}">
+                <oui-tile-definition data-term="{{:: 'pci_projects_project_streams_stream_status_label' | translate }}">
+                    <oui-tile-description>
+                        <span class="oui-status"
+                            data-ng-class="{
+                                'oui-status_success': $ctrl.stream.isRunning(),
+                                'oui-status_warning': $ctrl.stream.isInstalling(),
+                                'oui-status_error': $ctrl.stream.isError(),
+                            }">
+                            <span data-ng-bind="'pci_projects_project_instances_status_' + $ctrl.stream.status | translate"></span>
+                        </span>
+                    </oui-tile-description>
+                </oui-tile-definition>
+                <oui-tile-definition data-term="{{:: 'pci_projects_project_streams_stream_localisation_label' | translate }}">
+                    <oui-tile-description>
+                        <span data-ng-bind="$ctrl.CucRegionService.getTranslatedMicroRegion($ctrl.stream.region.region)"></span>
```

Comment:
Maybe it could make sense to add a region property to the `Stream` instance

Reply:
`Stream#region` contains the `GET .../region/:regionId` API call result, and it contains a `region` field :confused: